crates/stdarch-gen/neon.spec

```diff
@@ -7044,7 +7044,7 @@ a = 1.1, 1.9, -1.7, -2.3
 validate 1.0, 2.0, -2.0, -2.0
 target = frintts
 
-aarch64 = frint32x
+aarch64 = nop
```

Comment:
Why is this changed to nop? This is incorrect.

Reply:
Same as above. Changes have been cancelled.